### ISSUE TITLE
Camera.zoom() not working

### ISSUE THREAD

**Issue by danieljuschus** [edsaac/streamlit-PyVista-viewer#10]
Changing the value in `camera.zoom()` has no effect.

MWE:

```python
import pyvista as pv
from stpyvista import stpyvista

plotter = pv.Plotter(window_size=[400,400])
mesh = pv.Cube(center=(0,0,0))
plotter.add_mesh(mesh)
plotter.view_isometric()
plotter.camera.zoom(100)
stpyvista(plotter)
```

**Comment by edsaac** [comment]
Yes, for some reason `panel` does no use the camera zoom passed from `pyvista`. I believe the camera can be controlled with a separate dict as a kwarg of `pane.vtk` (https://panel.holoviz.org/api/panel.pane.vtk.html#panel.pane.vtk.vtk.AbstractVTK), but I've not tested this yet.

**Comment by m-froment** [comment]
Hello! Is there a way to zoom or control the projection (ex. using `enable_parallel_projection()` projection) now ?
